test/ttapp.py

```diff
@@ -140,3 +143,13 @@ def wait_data_files(tt_app, instances):
 
 def make_identical_subdir(tt_app):
     os.mkdir(tt_app.app_path(os.path.basename(tt_app.app_dir)))
+
+
+def remove_app_script(tt_app):
+    time.sleep(0.5)
```

Comment:
Why do we need this sleep with magic number?

Reply:
The quick-and-dirty way to make sure the script is removed after it have been read. The better way is using the file that is created during the instance script execution as an indicator that script have been read already. I considered the latter initially but forgot to switch to it finally.

Reply:
That was temporary solution. I'm working to drop it.

Reply:
`sleep` still exists.

Reply:
Yes, it was the last one to fix. Now it's done.